staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -315,23 +320,96 @@ func strategicPatchObject(
 
 type applyPatcher struct {
 	*patcher
+
+	model proto.Schema
 }
 
-func (p *applyPatcher) applyPatchToCurrentObject(currentObject runtime.Object) (runtime.Object, error) {
-	versionedObjToUpdate, err := p.creater.New(p.kind)
+// This is not going to stay, but we don't have a way to pass the
+// workflow id so far, so let's always use this one...
+const workflowId = "default"
+
+func (p *applyPatcher) LiveObject(obj runtime.Object) (map[string]interface{}, error) {
```

Comment:
Is there a reason for these methods to be public?

Reply:
(it could be useful to make them part of an interface for testing, but they still needn't be public?)

Reply:
No. I don't think it matters so much since the type is private anyway (or am I wrong?). I'll make them private anyway.

Reply:
public methods on private types are usually done when you're trying to implement some interface. Otherwise it's just confusing. And requires godoc comments, which you should maybe provide anyway ;)